test/test-pipe-bind-error.c

```diff
@@ -23,6 +23,7 @@
 #include "task.h"
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/un.h>
```

Comment:
This is causing the Windows CI to fail and should be conditional (maybe `#ifndef _WIN32`). 
https://github.com/libuv/libuv/actions/runs/10064572225/job/27821761240?pr=4442#step:3:843